docs/UsersGuide/source/CustomizingTheWorkflow/ConfigWorkflow.rst

```diff
@@ -635,48 +973,33 @@ For each workflow task, certain parameter values must be passed to the job sched
 ``EXTRN_MDL_NAME_LBCS``: (Default: "FV3GFS")
    The name of the external model that will provide fields from which lateral boundary condition (LBC) files (except for the 0-th hour LBC file) will be generated for input into the forecast model. Valid values: ``"GSMGFS"`` | ``"FV3GFS"`` | ``"GEFS"`` | ``"GDAS"`` | ``"RAP"`` | ``"HRRR"`` | ``"NAM"``
 
-``LBC_SPEC_INTVL_HRS``: (Default: "6")
+``LBC_SPEC_INTVL_HRS``: (Default: 6)
    The interval (in integer hours) at which LBC files will be generated. This is also referred to as the *boundary update interval*. Note that the model selected in ``EXTRN_MDL_NAME_LBCS`` must have data available at a frequency greater than or equal to that implied by ``LBC_SPEC_INTVL_HRS``. For example, if ``LBC_SPEC_INTVL_HRS`` is set to "6", then the model must have data available at least every 6 hours. It is up to the user to ensure that this is the case.
 
-``EXTRN_MDL_LBCS_OFFSET_HRS``: (Default: "")
-   Users may wish to use lateral boundary conditions from a forecast that was started earlier than the start of the forecast configured here. This variable indicates how many hours earlier the external model started than the FV3 forecast configured here. For example, if the forecast should use lateral boundary conditions from the GFS started 6 hours earlier, then ``EXTRN_MDL_LBCS_OFFSET_HRS: "6"``. Note: the default value is model-dependent and is set in ``ush/set_extrn_mdl_params.py``.
+``EXTRN_MDL_LBCS_OFFSET_HRS``: (Default: ``'{{ 3 if EXTRN_MDL_NAME_LBCS == "RAP" else 0 }}'``)
+   Users may wish to use lateral boundary conditions from a forecast that was started earlier than the start of the forecast configured here. This variable indicates how many hours earlier the external model started than the forecast configured here. For example, if the forecast should use lateral boundary conditions from a GFS forecast started six hours earlier, then ``EXTRN_MDL_LBCS_OFFSET_HRS: 6``. Note: the default value is model-dependent and is set in ``ush/set_extrn_mdl_params.py``.
 
 ``FV3GFS_FILE_FMT_LBCS``: (Default: "nemsio")
    If using the FV3GFS model as the source of the :term:`LBCs` (i.e., if ``EXTRN_MDL_NAME_LBCS: "FV3GFS"``), this variable specifies the format of the model files to use when generating the LBCs. Valid values: ``"nemsio"`` | ``"grib2"`` | ``"netcdf"``
 
-
 File and Directory Parameters
 --------------------------------
 
+``EXTRN_MDL_SYSBASEDIR_LBCS``: (Default: '')
```

Comment:
This one (and EXTRN_MDL_SYSBASEDIR_ICS) are another interesting case for a potential future issue. While this functionality did exist at one point, it has apparently been removed from all Tier 1 platforms, so now these variables just duplicate the function of `EXTRN_MDL_SOURCE_BASEDIR_LBCS` and `EXTRN_MDL_SOURCE_BASEDIR_ICS` respectively.

I think it's fine to keep this section as is for now but we should probably retire this variable in the future, unless the old functionality can be restored.

Reply:
Created Issue #922 for this.